python-sdk/src/astro/extractors/extractor.py

```diff
@@ -0,0 +1,55 @@
+from typing import Dict, List, Optional
+
+from airflow.models.taskinstance import TaskInstance
+from openlineage.airflow.extractors import TaskMetadata
+from openlineage.airflow.extractors.base import BaseExtractor
+from openlineage.airflow.utils import get_job_name
+from openlineage.client.facet import BaseFacet
+from openlineage.client.run import Dataset as OpenlineageDataset
+
+
+class OpenLineageFacets:
+    def __init__(
+        self,
+        inputs: List[OpenlineageDataset],
+        outputs: List[OpenlineageDataset],
+        run_facets: Dict[str, BaseFacet],
+        job_facets: Dict[str, BaseFacet],
+    ):
+        self.inputs = inputs
+        self.outputs = outputs
+        self.run_facets = run_facets
+        self.job_facets = job_facets
```

Comment:
Needs docstrings for it:

And this can be simplified as:

```python
@attr.define
class OpenLineageFacets:
        inputs: List[OpenlineageDataset]
        outputs: List[OpenlineageDataset]
        run_facets: Dict[str, BaseFacet]
        job_facets: Dict[str, BaseFacet]
```

Reply:
[Done](https://github.com/astronomer/astro-sdk/pull/1025/commits/7dad9a6fc4f0bcc5205f25fe51fb0b3f629a621e)

Reply:
Added the docstring: https://github.com/astronomer/astro-sdk/pull/1025/commits/82f23c37dd6e507b2ba3b6707ceb0674b9156207